evaluation/train.py

```diff
@@ -38,6 +40,12 @@ def main():
     if not eval_args.eval_tasks:
         raise ValueError("Must provide at least one eval task!")
 
+    if 'jigsaw_toxicity_pred' in eval_args.eval_tasks:
+        if eval_args.data_dir is None:
+            raise ValueError("Must provide data path for jigsaw_toxicity_pred")
```

Comment:
Can you please add the instruction from the README (ie "you have to download the dataset manually...") here as well, just so that folks don't have to then go look for instructions when this error is thrown?